src/Components/CreateImageWizard/formComponents/Packages.js

```diff
@@ -31,6 +33,32 @@ import api from '../../../api';
 import { useGetArchitecturesByDistributionQuery } from '../../../store/apiSlice';
 import isBeta from '../../../Utilities/isBeta';
 
+const ExactMatch = ({
+  pkgList,
+  search,
+  chosenPackages,
+  selectedAvailablePackages,
+  handleSelectAvailableFunc,
+}) => {
+  const match = pkgList.find((pkg) => pkg.name === search);
+  return (
+    <DualListSelectorListItem
+      data-testid={`exact-match-${match.name}`}
+      key={match.name}
```

Comment:
I think that we technically don't really need a key here, since this list item is not part of an array:
https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key

`Keys tell React which array item each component corresponds to, so that it can match them up later. This becomes important if your array items can move (e.g. due to sorting), get inserted, or get deleted. A well-chosen key helps React infer what exactly has happened, and make the correct updates to the DOM tree.`

Reply:
Yeah, I think you're right. I'll remove it.